src/strategies/hedgey-delegate/index.ts

```diff
@@ -0,0 +1,60 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'hedgeyfinance';
+export const version = '1.0.0';
+
+const abi = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'delegate',
+        type: 'address'
+      },
+      {
+        internalType: 'address',
+        name: 'token',
+        type: 'address'
+      }
+    ],
+    name: 'delegatedBalances',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: 'delegatedBalance',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.contract, 'delegatedBalances', [
+      address,
+      options.token
+    ])
+  );
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, balance]) => [
+      address,
+      parseFloat(formatUnits(balance, options.decimals)) * options.multiplier
+    ])
+  );
```

Comment:
hi @bark4mark I think you can use the `math` strategy with `contract-call` in it :) Can try?

Reply:
Would it be possible to have this strategy merged? I would like to keep the configuration minimal as I have to create documentation around voting and configuring the smart contracts.